network-api/networkapi/buyersguide/templates/buyersguide/catalog.html

```diff
@@ -0,0 +1,125 @@
+{% extends "../bg_base.html" %}
+
+{% load env cloudinary i18n static wagtailroutablepage_tags %}
```

Comment:
This is a copy of catalog.html – as to not use a template that's already in use by PNI products. 

The only change is the wagtailroutablepage_tags and the use of `{{ routablepageurl page 'product-view' product.slug }}` in the template